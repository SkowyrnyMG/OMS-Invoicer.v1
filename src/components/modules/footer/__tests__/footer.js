import React from 'react';

import {
  renderWithReduxRouter,
  screen,
  snapShotTest,
} from 'utils/tests/test-helper';
import Footer from '../footer';

describe('Footer', () => {
  test('should display in the document with correct styles', () => {
    snapShotTest(<Footer />);
  });

  test('anchor should have correct attributes', () => {
    renderWithReduxRouter(<Footer />);

    const anchor = screen.getByText('App created by Mateusz Gru≈∫la');

    expect(anchor).toHaveAttribute('href', 'https://mateuszgruzla.pl/');
  });
});
