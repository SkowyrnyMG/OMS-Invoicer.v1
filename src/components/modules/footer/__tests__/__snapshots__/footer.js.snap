// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Footer should display in the document with correct styles 1`] = `
.c0 {
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-box-pack: center;
  -webkit-justify-content: center;
  -ms-flex-pack: center;
  justify-content: center;
  -webkit-align-items: center;
  -webkit-box-align: center;
  -ms-flex-align: center;
  align-items: center;
  margin: 0;
  height: -webkit-fit-content;
  height: -moz-fit-content;
  height: fit-content;
  background: #446DF644;
}

<footer
  class="c0"
>
  <a
    href="https://mateuszgruzla.pl/"
    rel="noreferrer"
    target="_blank"
  >
    App created by Mateusz Gru≈∫la
  </a>
</footer>
`;
